python/lsst/pipe/tasks/makeCoaddTempExp.py

```diff
@@ -117,13 +117,19 @@ def run(self, patchRef, selectDataList=[]):
             except (KeyError, ValueError):
                 visitId = i
 
-            exp = self.createTempExp(calexpRefList, skyInfo, visitId)
-            if exp is not None:
+            exps = self.createTempExp(calexpRefList, skyInfo, visitId)
```

Comment:
I picked  a Struct over a dict, because I thought we had a rule about Tasks passing Structs instead of dicts.  I'll make the dataRefList appending and warp-writing generic regardless.